app/app.go

```diff
@@ -53,99 +43,42 @@ func New(options ...AppOption) *App {
 	return app
 }
 
-func (a *App) InitServer() {
-	a.srv.AppInitializedOnce.Do(func() {
```

Comment:
`AppInitializedOnce` is not used anymore. Should we remove it?

Reply:
Yeah definitely 👍